aha-table.html

```diff
@@ -21,14 +21,14 @@
     <content id="columndefs"></content>
 
     <template is="dom-if" if="{{showColumnChooser}}">
-      <div class="flex columnChooser u-mb-">
+      <div class$="{{_getColumnChooserClass(greedyHeightWithScroll)}}">
```

Comment:
Is it possible to write something like `<div class$="pagination {{_getColumnChooserClass(greedyHeightWithScroll)}}">` so the class names are in the template as much as possible? Applies to all other cases of this.